server/app.js

```diff
@@ -1,12 +1,14 @@
 import Koa from 'koa';
 import config from './config';
 import serve from 'koa-static';
+import compress from 'koa-compress';
 import {router} from './routes';
 import render from './view/render';
 import {enforceSSL} from './middleware/enforce-ssl';
 
 const app = new Koa();
 
+app.use(compress());
```

Comment:
Could we potentially be explicit about what we compress?
I only ask because of some reading I have been doing around [BREACH](http://security.stackexchange.com/questions/40562/are-there-any-risk-in-serving-static-assets-via-compressed-ssl-protocol-from-3rd).